base/constants.go

```diff
@@ -60,6 +60,19 @@ const (
 
 	DefaultViewQueryPageSize = 5000 // This must be greater than 1, or the code won't work due to windowing method
 
+	// Default the max number of idle connections per host to a relatively high number to avoid
+	// excessive socket churn caused by opening short-lived connections and closing them after, which can cause
+	// a high number of connections to end up in the TIME_WAIT state and exhaust system resources.  Since
+	// GoCB is only connecting to a fixed set of Couchbase nodes, this number can be set relatively high and
+	// still stay within a reasonable value.
+	DefaultHttpMaxIdleConnsPerHost = 256
```

Comment:
If we're always going to be converting these to strings, should we just define the constants as strings?

Reply:
I was originally going to make a point that it's easier to reuse if it's in int format, since most methods/structs will want an int value, but per YAGNI (ya aint gonna need it), I just converted to strings for now.